fdm-app/app/components/blocks/harvest/schema.ts

```
@@ -0,0 +1,30 @@
+import { z } from "zod"
+
+export const FormSchema = z.object({
+    b_lu_harvest_date: z.preprocess(
+        (arg) => {
+            if (typeof arg == "string" || arg instanceof Date)
```

Biome (2.1.2)

[error] Using == may be unsafe if you are relying on type coercion.

== is only allowed when comparing against null.
Unsafe fix: Use === instead.


(lint/suspicious/noDoubleEquals)